cfme/fixtures/pytest_selenium.py

```diff
@@ -242,9 +243,9 @@ def in_flight():
         The keys are: ``jquery, prototype, miq, spinner and document``.
         The values are: ``True`` if running, ``False`` otherwise.
     """
-    try:
+    with diaper:
         return execute_script(js.in_flight)
-    except:  # This diaper is OK (mfalesni)
+    if diaper.smelly:
```

Comment:
In fact, I'm going to make sure he's one of the authors, too. I'm taking the least amount of credit possible. :D